scripts/core/utils.js

```diff
@@ -3,3 +3,9 @@ export function stripHtmlTags(value) {
 
     return value ? String(value).replace(htmlRegex, '') : '';
 }
+
+export function getAngularService(name) {
+    return angular.element(document.body)
```

Comment:
Couldn't we use `ng.get` to get a service?

```js
import ng from './services/ng'
ng.get(name)
```

Reply:
```
import ng from 'core/services/ng';
console.log(ng);
console.log(ng.get('content'));
```

```
ProviderService {}
ng.js:48 Uncaught ng: $injector not registered for core/services/ng
```

Unfortunately not. You can only use `ng` when you have angular stuff already configured in that scope. It doesn't work outside angular for example in React components unless you pass some props all the way down from the place you are calling `ReactDOM.render`.

@petrjasek do you think it would be safe to replace the implementation of `scripts/core/services/ng.js` with my implementation of `getAngularService` except that I'd only return the injector and not the servivce itself?

Reply:
yep it will only work after angular config phase, but this should be the same with your implementation.
so this shouldn't work neither
```
import {getAngularService} from 'core/utils';
getAngularService('content');
```

Reply:
The code you posted does work. I'd guess it wouldn't work if angular hasn't finished it's config phase/bootstraping/mouting whatever it's called, but once it does - it works everywhere - which is why it's especially useful in React components as it allows accessing services without having to pass props 10 levels deep

Reply:
I think they both do the same thing, so if you do in your code:
```js
const getPromises = () => [ng.get('content').getCustomFields()];
```
it will work the same. it's only called when rendering so angular should be configured at that time

Reply:
I checked it and you are right, I'll use ng.